metaflow/plugins/aws/batch/batch.py

```diff
@@ -56,10 +58,10 @@ def _job_name(self, user, flow_name, run_id, step_name, task_id, retry_count):
         return '{user}-{flow_name}-{run_id}-{step_name}-{task_id}-{retry_count}'.format(
             user=user,
             flow_name=flow_name,
-            run_id=run_id,
+            run_id=str(run_id or ''),
```

Comment:
Small nit: this will create an empty string for a run ID of 0. Ditto for next two cases. Might be a bit problematic for retry count which may be more frequently 0?

Reply:
Good catch, I will modify the behaviour.